src/core/server/application/capabilities/merge_capabilities.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import typeDetect from 'type-detect';
+import { merge } from 'lodash';
+
+import { Capabilities } from '../../../public';
+
+export const mergeCapabilities = (...sources: Array<Partial<Capabilities>>): Capabilities =>
```

Comment:
There is `mergeCapabilities` on the client. I cannot find that someone used it. Could you clean this up, please?
https://github.com/elastic/kibana/blob/master/src/legacy/server/capabilities/merge_capabilities.ts

Reply:
This has already been removed, and will be switched over when the capabilities service changes land.